tests/hmm/hmumu_utils.py

```diff
@@ -387,11 +387,12 @@ def analyze_data(
     
     #Now actually call the JEC computation for each scenario
     jet_pt_startfrom = "pt_jec"
-    if is_mc and parameters["do_jer"]:
+    if is_mc and parameters["do_jer"][dataset_era]:
         jet_pt_startfrom = "pt_jec_jer"
     for uncertainty_name in syst_to_consider:
         #This will be the variated pt vector
         #print("computing variated pt for", uncertainty_name)
+        if(uncertainty_name=='jer'):continue
```

Comment:
can be removed - `do_jer` should take care of this.

Reply:
Done!